src/common/enums/getSignedUrlCommand.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum GetSignedUrlCommand {
+  PUT,
```

Comment:
you do not need to use PUT presignedUrl to upload images, 
You only need Get to generate resigned URL so you can return it to the client.